ddtrace/tracer/tracer.go

```diff
@@ -55,6 +55,39 @@ type tracer struct {
 
 	// pid of the process
 	pid string
+
+	// statsd client for tracer metrics.
+	statsd tracerStats
+}
+
+type tracerStats interface {
```

Comment:
Can we call this `statsdClient`? In and off itself, that's what it is. The context where it is used is not necessary in the name.